target/linux/mediatek/filogic/base-files/lib/preinit/10_fix_eth_mac.sh

```diff
@@ -8,6 +8,12 @@ preinit_set_mac_address() {
 		ip link set dev eth0 address "$addr"
 		ip link set dev eth1 address "$addr"
 		;;
+	mercusys,mr90x-v1)
+		addr=$(get_mac_binary "/tmp/tp_data/default-mac" 0)
+		ip link set dev eth1 address "$(macaddr_add $addr 1)"
```

Comment:
Can't this be placed in 02_network?

Reply:
Really, this can be placed in `02_network`. On the other side I deliberately place it here to have the correct MAC as early as possible. 

Unfortunately, this trick for some reason doesn't work for `eth0` so I had to override LAN MACs configuration through the `02_network`. But I would like to see `eth0` here too.